src/ott/geometry/geometry.py

```diff
@@ -129,30 +129,58 @@ def mean_cost_matrix(self) -> float:
     tmp = self._masked_geom().apply_cost(self._n_normed_ones).squeeze()
     return jnp.sum(tmp * self._m_normed_ones)
 
+  @property
+  def std_cost_matrix(self) -> float:
+    r"""Standard deviation of all values stored in :attr:`cost_matrix`.
+
+    Uses the :meth:`~ott.geometry.Geometry.apply_square_cost` to remain
```

Comment:
Should be just :meth:\`apply_square_cost\`; currently it doesn't render correctly.